.stignore

```diff
@@ -0,0 +1,56 @@
+.git
```

Comment:
For these samples, I would keep the `.stignore` file simple. 
I think we only need:
```
.git
node_modules
```